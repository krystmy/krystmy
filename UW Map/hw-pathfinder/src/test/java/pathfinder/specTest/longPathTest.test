#test finding a path

CreateGraph graph

AddNode graph A
AddNode graph B
AddNode graph C
AddNode graph D
AddNode graph E

AddEdge graph A B 1.0
AddEdge graph B C 2.0
AddEdge graph C D 3.0
AddEdge graph D E 4.0

FindPath graph A E