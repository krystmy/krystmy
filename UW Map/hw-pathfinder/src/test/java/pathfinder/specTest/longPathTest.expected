#test finding a path

created graph graph

added node A to graph
added node B to graph
added node C to graph
added node D to graph
added node E to graph

added edge 1.000 from A to B in graph
added edge 2.000 from B to C in graph
added edge 3.000 from C to D in graph
added edge 4.000 from D to E in graph

path from A to E:
A to B with weight 1.000
B to C with weight 2.000
C to D with weight 3.000
D to E with weight 4.000
total cost: 10.000